verification/helpers_test.go

```diff
@@ -0,0 +1,79 @@
+package verification
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"path/filepath"
+	"strconv"
+	"testing"
+)
+
+func TestGetArtifactDigestFromUri(t *testing.T) {
+
+	tests := []struct {
+		artifactUri string
+		digest      string
+		wantErr     bool
+	}{
+		{"domain.com:80/repository:digest", "digest", false},
```

Comment:
We should set `wantErr: true` for this case.

Reply:
Updated the code to accept `domain.com:80/repository@<alg>:<digest>`

My understanding is that <alg> is not included when querying manifests from the registry, is that correct? Or the entire thing `<alg>:<digest>` is considered a digest and we need to use that when [resolving](https://github.com/notaryproject/notation-go/blob/main/registry/interface.go#L27)?

Reply:
NVM. Alg is needed to query with a digest. I addressed it in the latest revision